src/components/badge/badge.scss

```diff
@@ -23,21 +23,27 @@
 
     font-size: functions.pxToRem(11);
 
-    height: functions.pxToRem(20);
-    line-height: 1;
-    min-width: functions.pxToRem(20);
-    padding: 0 functions.pxToRem(5);
+    max-height: 1.25rem;
```

Comment:
changing height to max-height makes badges with a label a few pixels smaller (20px to 16.5px). Is this intentional?

Reply:
I'm so happy that you noticed this! Yes it is intentional. I feel they were too clunky previously. Unnecessarily too big.

But you got me thinking, should I announce a BREAKING CHANGE in the commit message? Something like:

```
BREAKING CHANGE:
Badge height is decreased from `1.25rem` to `1rem`. 
This may have a slight effect on your UI design.
```

Reply:
No, unless I'm totally underestimating the impact here, I don't think that should be considered a breaking change.